nemo_text_processing/text_normalization/normalize_with_audio.py

```diff
@@ -42,6 +40,7 @@
 To run this script with a .json manifest file:
     python normalize_with_audio.py \
            --audio_data PATH/TO/MANIFEST.JSON \
+           --lang en \
```

Comment:
in the argparse it's called `language`, could you make sure the name consistent? maybe add a shorter version as an alternative name?